diagnostic.md

```diff
@@ -3,67 +3,72 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+element.style
+div
+body
+html
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+$( "OL" ).children().css();
+$( "OL" ).find( "li" ).css();
```

Comment:
This is almost right, be careful with capitalization. The `ol` should be lowercase. Also because you chained on the extra `.css()` it will return the css stylings of the element instead of the element itself.